pkg/network/domainspec/generators_test.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package network
+package domainspec
```

Comment:
Seems that `ctrl.Finish()` is still missing in the `AfterEach` of this file. Now after all the cleanup I think it is a good opportunity to add it.

Reply:
Done.